reporter/reporter.go

```diff
@@ -123,14 +124,27 @@ func ListenExecutionEvents() {
 			switch e.Topic {
 			case event.SpecStart:
 				r.SpecStart(e.Item.(*gauge.Specification).Heading.Value)
+				if e.Item.(*gauge.Specification).UsesArgsInContextTeardown(e.Item.(*gauge.Specification).DataTable.Table.Headers...){
```

Comment:
There is no need to store this info. A scenario has a `DataTableRow` field. This scenario field should only be assigned if it is a data table driven scenario and uses a dynamic param. If that field has some value then only print table info.